src/crypto/CHIPCryptoPALPSA.h

```diff
@@ -28,6 +28,7 @@
 
 #include "CHIPCryptoPAL.h"
 #include <lib/core/DataModelTypes.h>
+#include <lib/support/CHIPMem.h>
```

Comment:
Do we need it here?

Reply:
No, good catch, it's a leftover after something that I tried to do here before :)